<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelayerWdg</class>
 <widget class="QWidget" name="DelayerWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayoutDelay">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbDelay">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>延迟时间(ms):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbDelay">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letDelay">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatLineEdit</class>
   <extends>QLineEdit</extends>
   <header>XMatLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>XMatComboBox</class>
   <extends>QComboBox</extends>
   <header>XMatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
